### ISSUE TITLE
MEGU doesn't actually have a !Q output

### ISSUE THREAD

**Issue by msinger** [furrtek/DMG-CPU-Inside#53]
On page 32, the DFF `MEGU` is one of those DFFs that only have a `Q` output. In the schematic though, the `!Q` output is connected. This is wrong. It should be the `Q` output which is connected to `LALA` and `MYVY`. It should look like `MUKU` above or `NASA` below on the same page.

@rgalland, are you able to confirm that with your simulation?

**Comment by rgalland** [comment]
@msinger, yes you're right. For clarity there should be another symbol for the DFFs without /Q outputs (all of these are on page 32 and 33).
